app/assets/javascripts/govuk_publishing_components/analytics/page-content.js

```diff
@@ -14,7 +14,7 @@
       case isDocumentCollectionPage():
         return document.querySelectorAll('.document-collection .group-title').length
       case isMainstreamBrowsePage():
-        return countVisible(document.querySelectorAll('#subsection ul')) || document.querySelectorAll('#section ul').length || document.querySelectorAll('#root ul').length
+        return countVisible(document.querySelectorAll('#subsection ul')) || document.querySelectorAll('#section ul').length || document.querySelectorAll('#root ul').length || document.querySelectorAll('.gem-c-cards__list').length
```

Comment:
I'm wondering about this approach. Rather than extending the existing `isMainstreamBrowsePage` functions, would it be cleaner to create a whole new set that we can adapt as needed? Something like this.

```
PageContent.getNumberOfSections = function () {
...
  case isNewBrowsePage():
    return document.querySelectorAll('.gem-c-cards__list').length

...

PageContent.getNumberOfLinks = function () {
...
  case isNewBrowsePage():
      document.querySelectorAll('.gem-c-cards__link').length

...

  function isNewBrowsePage () {
    return getMetaAttribute(metaApplicationSelector) === 'collections' &&
      getMetaAttribute(metaFormatSelector) === 'new_browse_page'
  }
```

Obviously it will involve more code changes but it might be more flexible if we need to adapt it, and easier to clean up if we retire the existing browse page code.

Reply:
I'm open to do this - it isolates it more and gives more granular control for these new templates. 

Thinking out loud, it would be nice for this to exist in the component - or the page - and pass into tracking.

Reply:
Yeah, there's definitely some things about how this tracking works that aren't ideal - having this script hidden away separate from the browse pages that explicitly relies upon the markup structure of those pages (again) feels brittle. But hopefully we can address issues like this in the upcoming GA work.